README.md

```diff
@@ -387,21 +387,46 @@ echo -n baz | kubectl create secret generic mysecret --dry-run=client --from-fil
 
 Creating temporary Secret with the `kubectl` command, only to throw it away once piped to `kubeseal` can
 be a quite unfriendly user experience. We're working on an overhaul of the the CLI experience. In the meantime,
-we offer an alternative mode where kubeseal only cares about encrypting a value to stdout and it's your responsibility to put it inside a SealedSecret resource (not unlike any of the other k8s resources).
+we offer an alternative mode where kubeseal only cares about encrypting a value to stdout and it's your responsibility to put it inside a `SealedSecret` resource (not unlike any of the other k8s resources).
 
 It can also be useful as a building block for editor/IDE integrations.
 
 The downside is that you have to be careful to be consistent with the sealing scope, the namespace and the name.
-See [Scopes](#scopes):
 
-```console
+See [Scopes](#scopes)
+
+#### strict scope (default)
```

Comment:
Could you please run [doctoc](https://github.com/thlorenz/doctoc) to regenerate the TOC? You're including new headers that should be taken into account.
Thanks in advance

Reply:
Thanks. I hadn't considered the TOC. This would add a 3rd level to the TOC and these don't really need to be headings. Updated to use formatting more similar to the `Scopes` section.